opm/simulators/wells/BlackoilWellModel.hpp

```diff
@@ -384,10 +385,15 @@ namespace Opm {
             // and in the well equations.
             void assemble(const int iterationIdx,
                           const double dt);
-            bool assembleImpl(const int iterationIdx,
-                              const double dt,
-                              const std::size_t recursion_level,
-                              DeferredLogger& local_deferredLogger);
+
+            // the function handles one itearation of updating well controls and also network related.
```

Comment:
The comment is a bit unclear, would it be correct to say "updating well controls and network pressures"?

Typo: itearation->iteration

Reply:
I rewrote some of the comments here, hopefully, it is easier to understand.